content/en/docs/about/community.md

```diff
@@ -103,6 +103,22 @@ In addition, please check out the community-curated [awesome list of projects an
 
 The Kubeflow trademark and logos are registered trademarks of Google, please review the [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf) for more information.
 
+## Kubeflow Steering Committee
+
+The [Kubeflow Steering Committee (KSC)](https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md) is the governing body of the Kubeflow project, providing decision-making and oversight pertaining to the Kubeflow project policies, sub-organizations, and financial planning, and defines the project values and structure.
```

Comment:
I think the table is duplicated from KUBEFLOW-STEERING-COMMITTEE.md? If so maybe a reference to the file would be sufficient.

Reply:
Good idea. Removed.